demo/tests/test_pipeline.py

```diff
@@ -32,12 +32,24 @@ def test_dag_with_mock_sample_input_data(
 
         # Then
         try:
-            assert expected_staged_foo_df.equals(pd.read_parquet(raw_config.get(source_key="STAGED_FOO")["local"]["file_path"]))
-            assert expected_staged_bar_df.equals(pd.read_parquet(raw_config.get(source_key="STAGED_BAR")["local"]["file_path"]))
-            assert expected_final_foo_df.equals(pd.read_parquet(processed_config.get(source_key="FINAL_FOO")["local"]["file_path"]))
-            assert expected_final_bar_df.equals(pd.read_parquet(processed_config.get(source_key="FINAL_BAR")["local"]["file_path"]))
+            pd.testing.assert_frame_equal(
+                expected_staged_foo_df,
+                pd.read_parquet(raw_config.get(source_key="STAGED_FOO").local.file_path)
+            )
+            pd.testing.assert_frame_equal(
+                expected_staged_bar_df,
+                pd.read_parquet(raw_config.get(source_key="STAGED_BAR").local.file_path)
+            )
+            pd.testing.assert_frame_equal(
+                expected_final_foo_df,
+                pd.read_parquet(processed_config.get(source_key="FINAL_FOO").local.file_path)
+            )
+            pd.testing.assert_frame_equal(
+                expected_final_bar_df,
+                pd.read_parquet(processed_config.get(source_key="FINAL_BAR").local.file_path)
+            )
```

Comment:
Can you please configure pycharm to not mess with reformatting the line-length? Much readable if in one line

Reply:
This is `black` and it already aims for 120 character lines.